src/SharpGLTF.Cesium/Schema2/Ext.StructuralMetadataRoot.cs

```diff
@@ -326,7 +334,12 @@ public partial class PropertyAttributeProperty
         public string Attribute
         {
             get { return _attribute; }
-            set { _attribute = value; }
+            set
+            {
+
+                if (value == null) { _attribute = null; return; }
+                _attribute = value;
+            }
```

Comment:
Are you sure that's all right?

if value is null, _attribute is also null, so there's no need for the check, it was fine the way it was

Reply:
ah yes, removed them again